pyproject.toml

```diff
@@ -46,9 +45,6 @@ dev = [
 [tool.uv]
 required-version = "==0.8.*"
 
-[tool.uv.sources]
-dc-logging-utils = { path = "." }
```

Comment:
This is a bit of clean up. 

It's not actually necessary for this to be in the aws group, and including it means `uv sync` fails when we bump the version number. 

`uv sync`, `uv sync --all-groups` and `uv sync --group aws` all install the `dc-logging-utils` package. From another repo installing `dc-logging-utils` from a github revision or local directory doesn't install extra dependencies.